### PR TITLE
GameDB: Conflict 4, K-1 WorldGP 2006 and Mahou Sensei Negima! Kagai Jugyou fixes.

### PR DESCRIPTION
**Pull request by atomic83GitHub**
This PR add an emotion engine rounding fix for Conflict 4 which solve bad character behaviour, 2 VU Xgkick timing adjustment for K-1 World GP 2006 which solve missing colors on the fighters, 1 VU Xgkick timing adjustment for Mahou Sensei Negima! Kagai Jugyou which solve bad rendering, comment adjustment for Socom 2 fixes and a missing entry.

***Conflict 4 screenshots***

Without the fix:
capture

With the fix:
capture13

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -1765,8 +1765,8 @@ Serial = SCES-51904
 Name   = SOCOM II - U.S. Navy SEALs
 Region = PAL-M5
 Compat = 5
-EETimingHack = 1
-VIF1StallHack = 1// HUD.
+EETimingHack = 1	// Get rids of DMA8/9 non-fatal errors.
+VIF1StallHack = 1	// Fixes broken HUD.
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Tab space, adjust the others in the PR as well.

Should look like this
```
EETimingHack = 1 // Get rids of DMA8/9 non-fatal errors.
VIF1StallHack = 1 // Fixes broken HUD.
```

Comment (atomic83GitHub) [reply bin/GameIndex.dbf]: Done

**Comment by lightningterror** [comment]
Someone should test the other regions and add the fixes for those as well but that can be done in the future.

**Comment by atomic83GitHub** [comment]
Also, even with the fixes, Socom is still spamming this error quite often:

***989snd Error: cause 74 -> 0, 0, 0, 0***

**Pull request merged**